### PR TITLE
Switch from Celluloid to concurrent-ruby

### PR DESCRIPTION
**Pull request by qsoftdevelopment**
Switch from Celluloid to concurrent-ruby

### CODE REVIEW

File: lib/pubnub/client.rb [hunk 1]
```diff
@@ -320,27 +319,23 @@ def create_state_pools(event)
...
+           else
+             HTTPClient.new
+           end
 
       case event_type
       when :subscribe_event
-        hc.connect_timeout = @env[:s_open_timeout]
-        hc.send_timeout = @env[:s_send_timeout]
-        hc.receive_timeout = @env[:s_read_timeout]
+        hc.receive_timeout = 310
```
Comment (callumj) [inline lib/pubnub/client.rb]: Is there a reason these were removed? We'd like to have them restored so we can control the worst case operating time of these calls.

Comment (qsoftdevelopment) [reply lib/pubnub/client.rb]: @callumj Let me do a research about it and I will get back to you.

Comment (qsoftdevelopment) [reply lib/pubnub/client.rb]: @callumj I think this will do the things right for you https://github.com/pubnub/ruby/pull/72 ?

**Comment by captainkerk** [comment]
Hey there, can you make a new release with this change in it?

**Comment by qsoftdevelopment** [comment]
@captainkerk Hi, it is in the process.

**Comment by bitbier** [comment]
Any updates on a release here?

**Comment by qsoftdevelopment** [comment]
@bitbier Yes, we are in the final testing phase. So I expect to do the merge very soon :).

**Comment by captainkerk** [comment]
someone cut a new version back in August: https://rubygems.org/gems/pubnub/versions/4.0.28

That one doesnt have the celluloid dependency and worked for me.

**Pull request merged**